src/Config.php

```diff
@@ -19,22 +19,19 @@
  */
 class Config implements ConfigInterface
 {
-    protected $name;
-    protected $description;
-    protected $finder;
-    protected $dir;
```

Comment:
`$dir` is no longer used, so can be removed
changing the other from `protected` to `private` makes the code easier to maintain (i.e. less BC to worry about)